### PR TITLE
sync spl changes

### PR DESCRIPTION
**Pull request by samuelvanderwaal**

### CODE REVIEW

File: Anchor.toml [hunk 1]
```diff
@@ -2,28 +2,43 @@
...
 [provider]
 cluster = "localnet"
-wallet = "~/.config/solana/id.json"
+wallet = "./devkey.json"
 
 [scripts]
-test = "cargo test"
-update_idl = "cp -r ./target/idl ./src && cp -r ./target/types ./src"
+test = "pnpm jest"
+copy_idl = "cp ./target/types/m2.ts ./src/services/m2.ts"
```
Comment (madergaser) [inline Anchor.toml]: probably want to revert this change back and update the IDL at https://github.com/magicoss/m2/blob/main/src/idl/m2.json

Comment (samuelvanderwaal) [reply Anchor.toml]: Good catch

**Review by madergaser — APPROVED** [review]
